vcsutils/webhookparser/gitlab.go

```diff
@@ -54,10 +54,14 @@ func (webhook *GitLabWebhook) parseIncomingWebhook(payload []byte) (*WebhookInfo
 }
 
 func (webhook *GitLabWebhook) parsePushEvent(event *gitlab.PushEvent) *WebhookInfo {
+	var ts int64
```

Comment:
I suggest using a more informative variable name, for example:
```suggestion
	var localTimestamp int64
```